src/components/DataProvider/index.js

```diff
@@ -105,6 +115,14 @@ export default class DataProvider extends Component {
         await this.fetchData(id, 'UPDATE_SUBDOMAIN');
       }
     });
+
+    // Check if basedomain changed in props -- if so reset state.
```

Comment:
Should this cancel the live loading? And if `updateInterval` changed, then `fetchInterval` needs to be canceled and recreated

Reply:
Good catch -- fixed.